microsoft-edge/webview2/how-to/runtime-channel.md

```diff
@@ -0,0 +1,188 @@
+
+1.  Replace `C:\Users\myname\AppData\Local\Microsoft\Edge SxS\Application\88.0.680.0` by the path to the desired Runtime channel.<!--should this instead be "Microsoft Edge channel"?-->
+
+
+### Resuming using the default, WebView2 Evergreen Runtime
+
+To undo the above setting, run the following command:
+
+`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\BrowserExecutableFolder /f`
+
+
+<!--========================================================================-->
+## Using an environment variable
+
+If you use the environment-variable approach, you cannot specify a specific runtime path, but rather, you reverse the search order for the Runtime channel.<!-- "runtime channel" terminology ok?  "WebView2 Runtime or Microsoft Edge channel"?-->
+
+The default channel-search order when using the above, registry-override approach is:
+1.  The WebView2 Runtime.
+1.  The Beta channel of Microsoft Edge.
+1.  The Dev channel of Microsoft Edge.
+1.  The Canary channel of Microsoft Edge.
+
+If you instead use the environment-variable approach, that order is reversed.  So the channel-search order when using this environment-variable approach is:
+1.  The Canary channel of Microsoft Edge.
+1.  The Dev channel of Microsoft Edge.
+1.  The Beta channel of Microsoft Edge.
+1.  The WebView2 Runtime.
+
+<!-- how does the above info connect to the below info? -->
+
+To make your application use the Canary or Dev channel of Microsoft Edge by using an environment variable:
+
+1.  In the Windows search bar, enter "environment", and then select **Edit the system environment variables**.
+
+    :::image type="complex" source="./media/search-bar-edit-sys-env-vars.png" alt-text="Using the Windows search bar to find where to edit environment variables" lightbox="./media/search-bar-edit-sys-env-vars.png":::
+       Using the Windows search bar to find where to edit environment variables
+    :::image-end:::  
+
+1.  In the **System Properties** dialog box, select the **Advanced** tab, and then select the **Environment Variables** button.
+
+    :::image type="complex" source="./media/system-properties-env-vars.png" alt-text="The Environment Variables button in the System Properties dialog box" lightbox="./media/system-properties-env-vars.png":::
+       The **Environment Variables** button in the **System Properties** dialog box
+    :::image-end:::  
+
+1.  In the **User variables** section of the **Environment Variables** dialog box, select **New**.
+
+1.  In the **New User Variable** dialog box, set the **Variable name** to `WEBVIEW2_RELEASE_CHANNEL_PREFERENCE`, and set the **Variable value** to `1`.  Then select **OK** to close the dialog boxes.
+
+    :::image type="complex" source="./media/env-vars-new-user-variable.png" alt-text="Adding a new environment variable, as a user variable" lightbox="./media/env-vars-new-user-variable.png":::
+       Adding a new environment variable, as a user variable
+    :::image-end:::  
+
```

Comment:
After setting an environment variable this way it applies to new processes created by explorer. It doesn't apply to already running processes. You'd likely need to restart VS or log out and log back in to ensure everything picks up the new environment variable.